getinsomnia/insomnia
# feat(pre-req): enable auth manipulation through insomnia.request - INS-3379

Changes:
- [x] enabled auth manipulation through insomnia.request
- [x] added unit tests for auth transforming
- [x] added smoke tests for auth manipulation

### How to test
Run this script and check if it works:

```javascript
insomnia.request.auth.update(
    {
        type: 'basic',
        basic: [
                {key: 'username', value: 'myName'},
                {key: 'password', value: 'myPwd'},
        ],
    },
    'basic'
);
```
